### DISCUSSION TITLE
TestFlight Exception List

### DISCUSSION THREAD

**Discussion by Haroldfromk** [SwiftBreakers/TTEOPPOKKI4U#127] [General]
여기에 적어주세요 사진첨부하면 더 좋을것같아요

**Comment by Haroldfromk** [comment]
1. 카메라, 사진 업로드 안되는 이슈 [해결] 
- 진문님 : picker.delegate 추가를 함으로써 해결.

**Comment by labydin** [comment]
1. 리뷰 작성하기 페이지에서 취소버튼 누르면 가게 정보 페이지 네비게이션바 사라지는 이슈 -> 수정
2. 가게 정보 페이지 네비게이션 다크모드 컬러 이슈 -> 수정
3. 지도 화면 dismiss 시 네비게이션바 생성 이슈 -> 수정
4. 현재 위치 더블 클릭시 어플 다운 이슈 -> 내일 회의 때 한 번 더 확인하기
